DSCResources/MSFT_xADRecycleBin/Docs/readme.md

```diff
@@ -0,0 +1,15 @@
+The xADRecycleBin DSC resource will enable the Active Directory Recycle Bin feature for the target forest.
```

Comment:
Should I change that, since it as the root of the module?  I thought you would own that part.  I can update it if you like.  I just did not see all of the other resources in the module documented there.